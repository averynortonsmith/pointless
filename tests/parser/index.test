
x = foo[0]
y = foo + {bar: baz}[0]