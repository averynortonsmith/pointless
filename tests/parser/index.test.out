(Program 
 null
 []
 (Def 
  (Name "x")
  (Index 
   (Name "foo")
   (Number 0)))
 (Def 
  (Name "y")
  (BinaryOp 
   Tok.Add
   (Name "foo")
   (Index 
    (Dict 
     (Pair 
      (Name "bar")
      (Name "baz")))
    (Number 0)))))
